packages/googlecloud/functions/getanswer/inquirer.py

```diff
@@ -26,8 +26,8 @@
 from langchain_openai import OpenAIEmbeddings
 
 
-from helper import sort_retrieved_documents
-from api import RESPONSE_TYPE_DEPTH, RESPONSE_TYPE_GENERAL
+from .helper import sort_retrieved_documents
+from .api import RESPONSE_TYPE_DEPTH, RESPONSE_TYPE_GENERAL
```

Comment:
had to tweak the local imports - since it gets called from the deepeval library technically when the evaluation functions are run